autogen.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -e
+
+: "${AUTORECONF:=autoreconf}"
+
+AUTORECONFVERSION=`$AUTORECONF --version 2>&1 | grep "^autoreconf" | sed 's/.*) *//'`
+
+# The minimum required version of autoconf is currently 2.69.
+case "$AUTORECONFVERSION" in
+'' | 0.* | 1.* | 2.[0-5]* | 2.6[012345678]* )
```

Comment:
This would detect `2.100` and `2.600` as too old.

Reply:
Fixed.

Reply:
`2.100` still stands. It might be more convenient to split the version into major and minor using `cut` and then to flip from shell pattern space into shell arithmetic space:
```
if [ "$maj" -lt 2 ] || [ "$maj" -eq 2 -a "$min" -lt 69 ]; then
```

Reply:
Right, done.